Production/govuk_ios/ViewModels/SettingsViewModel.swift

```diff
@@ -100,19 +103,36 @@ class SettingsViewModel: SettingsViewModelInterface {
         let rowTitle = String.settings.localized(
             "helpAndFeedbackSettingsTitle"
         )
+        let feedbackUrl = helpAndFeedbackUrl?.absoluteString ?? Constants.API.helpAndFeedbackUrl
```

Comment:
this object creation is not needed, just return it below

Reply:
The line length would've been too long / complex, as we'd need to add `self`. However, this comment is no longer valid as a different solution mitigates the local var: https://github.com/alphagov/govuk-mobile-ios-app/pull/211/commits/a64302852dd3462a1549dfd7ef4a6f4a4f35c7cf.